src/rime/lever/deployment_tasks.cc

```diff
@@ -639,12 +639,14 @@ bool CleanOldLogFiles::Run(Deployer* deployer) {
       continue;
     vector<path> files;
     DLOG(INFO) << "temp directory: " << dir;
+    string app_name = deployer->distribution_name;
```

Comment:
[app_name](https://github.com/rime/librime/blob/ec2d87df5a23aa03cef3e46d03c490d87b1e5234/src/rime_api.h#L85) is used to name log files.

Reply:
> [app_name](https://github.com/rime/librime/blob/ec2d87df5a23aa03cef3e46d03c490d87b1e5234/src/rime_api.h#L85) is used to name log files.

yes，then check file_name starts_with app_name to ensure it's log files of app